### DISCUSSION TITLE
How to release source code to "Windows 64-bit installer" or "Windows 64-bit standalone" or other platform?

### DISCUSSION THREAD

**Discussion by zearoby** [qutebrowser/qutebrowser#6284] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/22544491/111110278-33c47100-8597-11eb-8782-9bf0c44f7511.png)
Would you mind show the user guide about release from source code? Thank you so much.

**Comment by The-Compiler** [comment, marked as answer]
I'm not sure what you're asking exactly - are you asking how you can build those files from the sources?

If so, see the [release checklist](https://github.com/qutebrowser/qutebrowser/blob/master/doc/contributing.asciidoc#qutebrowser-release). The main thing you'll need is something like `py -3.9 -m tox -e build-release -- --asciidoc $env:userprofile\\bin\\asciidoc-9.1.0\\asciidoc.py`.
